web/src/beta/lib/reearth-ui/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC, useCallback, useEffect, useState, ChangeEvent } from "react";
+
+import { fonts, styled } from "@reearth/services/theme";
+
+export type DatePickerProps = {
+  value?: string;
+  disabled?: boolean;
+  size?: "normal" | "small";
+  onChange?: (text: string) => void;
+  onBlur?: (text: string) => void;
+};
+
+export const DatePicker: FC<DatePickerProps> = ({ value, disabled, onChange, onBlur }) => {
+  const [currentValue, setCurrentValue] = useState(value ?? "");
+  const [isFocused, setIsFocused] = useState(false);
+
+  useEffect(() => {
+    setCurrentValue(value ?? "");
+  }, [value]);
+
+  const handleChange = useCallback(
+    (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+      const newValue = e.currentTarget.value;
+      setCurrentValue(newValue ?? "");
+      onChange?.(currentValue);
+    },
+    [currentValue, onChange],
+  );
+
+  const handleBlur = useCallback(() => {
+    setIsFocused(false);
+    onBlur?.(currentValue);
+  }, [currentValue, onBlur]);
+
+  const handleFocus = useCallback(() => {
+    setIsFocused(true);
+  }, []);
+
+  return (
+    <Wrapper status={isFocused ? "active" : "default"}>
+      <StyledInput
+        value={currentValue}
+        placeholder="MM/DD/YYYY"
```

Comment:
I find it been displayed in lowercase in storybook, could you investigate it? I think it should display in uppercase as it defined here.

Reply:
@airslice since we are using native html date picker the provide their own placeholder which I can not override it. so I think this placeholder is not used may be I can remove it.

Reply:
Emmm i think it's okey for now.